corehq/ex-submodules/couchforms/models.py

```diff
@@ -224,6 +224,27 @@ def metadata(self):
 
         return None
 
+    def set_meta_properties(self):
```

Comment:
could put this in `AbstractXFormInstance`

Reply:
Moved.